export.py

```diff
@@ -189,11 +226,11 @@ def combine(all_posts, all_comments):
         save_as_html(id, subfolder, json_post, post_comments_html)
         save_as_markdown(id, subfolder, json_post, post_comments_html)
 
-
 if __name__ == '__main__':
     if True:
-        all_posts = download_posts()
-        all_comments = download_comments()
+        # modified these calls to include cookies and headers since they can't be imported from auth.py anymore
```

Comment:
```suggestion
        # modified these calls to include cookies and headers since they can't be imported from auth.py anymore
```
This one is also helpful as a comment in the PR but not in the resulting code

Reply:
Done!